### PR TITLE
Delegate safeinputs dns

### PR DESCRIPTION
**Pull request by sleepycat**
Safeinputs is an innovation project at PHAC. As the name implies it's
aiming to safely handle inputs and that safety starts with encryption,
which requires a domain name... hence this PR.

### CODE REVIEW

File: terraform/safeinputs-alpha-canada-ca.tf [hunk 1]
```diff
@@ -0,0 +1,25 @@
+resource "aws_route53_record" "tracker-alpha-canada-ca-NS" {
```
Comment (maxneuvians) [inline terraform/safeinputs-alpha-canada-ca.tf:1]: The resources need different names

File: terraform/safeinputs-alpha-canada-ca.tf [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  records = [
+    "ns-cloud-e1.googledomains.com.",
+    "ns-cloud-e2.googledomains.com.",
+    "ns-cloud-e3.googledomains.com.",
+    "ns-cloud-e4.googledomains.com.",
+  ]
+  ttl = "300"
+}
+
+resource "aws_route53_record" "suivi-alpha-canada-ca-NS" {
```
Comment (maxneuvians) [inline terraform/safeinputs-alpha-canada-ca.tf:14]: see above

File: terraform/safeinputs-alpha-canada-ca.tf (hunk 1 above)
Comment (sleepycat) [reply terraform/safeinputs-alpha-canada-ca.tf]: 🤦
Fixed.

File: terraform/safeinputs-alpha-canada-ca.tf (hunk 2 above)
Comment (sleepycat) [reply terraform/safeinputs-alpha-canada-ca.tf]: Fixed in follow up commit.

**Comment by CalvinRodo** [comment]
Reviewing in PR #255. 

It's needed as this PR is from an external contributor and so has no access to secrets.

**Pull request closed without merging**